index.js

```diff
@@ -1,55 +1,333 @@
+// HTML SELECTORS
+const itemUl = document.querySelector('.store--item-list')
```

Comment:
Maybe put these under the state array so you can read them clearer when they appear in the code

Reply:
"These" being the consts